src/shared/components/carousel/arrow/index.module.css

```diff
@@ -0,0 +1,91 @@
+@import "shared/styles/imports/variables.css";
+
+.container {
+  position: absolute;
+  top: 50%;
+  width: 30px;
```

Comment:
Variables, variables, variables!

But as we are pressured by time let's do this later 😉

Reply:
We should do another pass of this file when we have the time.